openjdk/build.xml

```diff
@@ -197,7 +198,7 @@
 					<else>
 						<if>
 							<and>
-								<contains string="${env.SPEC}" substring="arm" />
+								<equals arg1="${PLATFORM}" arg2="arm_linux" />
```

Comment:
This is the only example I could find where the hardware component of SPEC was being used. That said, I'm pretty sure this branch never worked properly, as I wouldn't expect `env.SPEC` to resolve properly. The only `arm` SPEC was for the `arm_linux` PLATFORM so I figured this was an equivalent substitution.